### PR TITLE
check against PHP 7.0 (and nightly)

### PR DESCRIPTION
**Pull request by mutec**
nightly is allowed to fail in case the nightly build is buggy

### CODE REVIEW

**Comment by TimWolla** [comment]
Please leave out `nightly` and replace `5.6` by `7.0`. We're only doing syntax checks anyway. There is no point in taking up Travis' resources.

**Comment by mutec** [comment]
@TimWolla done.

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
 before_install:
-  - pyrus install pear/PHP_CodeSniffer
+  - pear install pear/PHP_CodeSniffer
   - phpenv rehash
 script:
   - find . -type f -name '*.php' -not -path './wcfsetup/install/files/lib/system/api/composer/autoload_static.php' -print0 |xargs -0 -I file -P 4 php -l file > /dev/null
   - phpcs -p --extensions=php --standard="`pwd`/CodeSniff/WCF" .
+matrix:
+  allow_failures:
+  - php: nightly
```
Comment (TimWolla) [inline .travis.yml]: This is obsolete now.

Comment (mutec) [inline .travis.yml]: Ups, forgot that.

**Comment by TimWolla** [comment]
Thanks.

**Pull request merged**